flink-tests/src/test/java/org/apache/flink/test/checkpointing/UnalignedCheckpointTestBase.java

```diff
@@ -773,7 +774,9 @@ public Configuration getConfiguration(File checkpointDir) {
             conf.set(
                     NettyShuffleEnvironmentOptions.NETWORK_BUFFERS_PER_CHANNEL, BUFFER_PER_CHANNEL);
             conf.set(NettyShuffleEnvironmentOptions.NETWORK_REQUEST_BACKOFF_MAX, 60000);
-            conf.setString(AkkaOptions.ASK_TIMEOUT, "1 min");
```

Comment:
The `ASK_TIMEOUT` is already at 1 minute. How long did it take for the test to fail? Are we really sure that it is the `ASK_TIMEOUT` which causes the test instability? Is there some other timeout which might cause it?